src/main/java/de/dhbw/karlsruhe/students/mailflow/external/infrastructure/email/CreateEmailHelper.java

```diff
@@ -15,11 +16,16 @@ private CreateEmailHelper() {}
 
   public static Email createEmailWithMessage(Message message) throws EmailParsingException {
     try {
-      return Email.create(message.getContent().toString(),
-          EmailMetadataFactory.withMessage(message).build(), null);
-    } catch (IOException | MessagingException e) {
+
+      String content = message.getContentType();
```

Comment:
```suggestion
      String content = message.getContent().toString()
```
Was this by accident or did I miss something?

Reply:
sonarlint displayed me that getContentType return already a String

Reply:
Yes because the contentType is a string. But previously we wrote getContent instead of getContentType